### PR TITLE
Use keywords structure in all places - `obfuscateQueries.cpp` / `Suggest.cpp`

### PR DESCRIPTION
**Pull request by nikitamikhaylov**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Use `system.keywords` to fill in the suggestions and also use them in the all places internally.

### CODE REVIEW

**Comment by robot-ch-test-poll1** [comment]
*This is an automated comment for commit d18aba1194cb1a48560315d420404783b418e439 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/62000/d18aba1194cb1a48560315d420404783b418e439/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: src/Parsers/obfuscateQueries.cpp [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#include "Parsers/CommonParsers.h"
```
Comment (nickitat) [inline src/Parsers/obfuscateQueries.cpp:1]:
```suggestion
#include <Parsers/CommonParsers.h>
```

File: src/Parsers/obfuscateQueries.cpp [hunk 2]
```diff
@@ -24,413 +27,50 @@ namespace ErrorCodes
...
-    "WHEN",
-    "WHERE",
-    "WINDOW",
-    "WITH",
-    "WK",
-    "WRITABLE",
-    "YEAR",
-    "YYYY",
-    "ZKPATH"
+    static std::unordered_set<std::string_view> instance;
```
Comment (nickitat) [inline src/Parsers/obfuscateQueries.cpp:32]: maybe

```suggestion
    static std::unordered_set<std::string_view> instance = initialize();
```

and remove `initialized`

Comment (alexey-milovidov) [inline src/Parsers/obfuscateQueries.cpp:32]: You could write an assignment to a lambda that you instantly call.

**Review by nickitat — APPROVED** [review]

File: src/Client/Suggest.cpp [hunk 3]
```diff
@@ -82,6 +60,7 @@ static String getLoadSuggestionQuery(Int32 suggestion_limit, bool basic_suggesti
     add_column("name", "data_type_families", false, {});
     add_column("name", "merge_tree_settings", false, {});
     add_column("name", "settings", false, {});
+    add_column("keyword", "keywords", false, {});
```
Comment (azat) [inline src/Client/Suggest.cpp:63]: This breaks completion for older servers, can we add this to the query only if the reversion/version is new enough?

Comment (nikitamikhaylov) [reply src/Client/Suggest.cpp:63]: Sure.

**Pull request merged**